src/app/public/modules/lookup/lookup.component.ts

```diff
@@ -162,19 +169,25 @@ export class SkyLookupComponent
   }
 
   public onTokensKeyUp(event: KeyboardEvent) {
-    const key = event.key.toLowerCase();
-    if (key === 'backspace') {
-      this.sendTokensMessage(SkyTokensMessageType.RemoveActiveToken);
-      this.sendTokensMessage(SkyTokensMessageType.FocusPreviousToken);
-      event.preventDefault();
-    }
-
-    if (key === 'delete') {
-      this.sendTokensMessage(SkyTokensMessageType.RemoveActiveToken);
-      this.windowRef.nativeWindow.setTimeout(() => {
-        this.sendTokensMessage(SkyTokensMessageType.FocusActiveToken);
-      });
-      event.preventDefault();
+    if (event.key) {
```

Comment:
To make this consistent with the autocomplete component, I added the null check and then converted this logic to a `switch` statement.